cloud/aws/deploy/website.go

```diff
@@ -218,9 +221,6 @@ func (a *NitricAwsPulumiProvider) deployCloudfrontDistribution(ctx *pulumi.Conte
 					Cookies: &cloudfront.DistributionOrderedCacheBehaviorForwardedValuesCookiesArgs{
 						Forward: pulumi.String("none"),
 					},
-					Headers: pulumi.StringArray{
```

Comment:
This shouldn't be removed. It is used in the url rewrite function for sub sites during cache invalidation. Maybe we need a comment.

Reply:
Yep, rebase error will fix up